{"kind":"symbol","schemaVersion":{"minor":3,"major":0,"patch":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/ciphertextsize(value:)"]}],"metadata":{"symbolKind":"case","role":"symbol","roleHeading":"Case","title":"Exception.cipherTextSize(value:)","modules":[{"name":"SwiftKyber"}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"value"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:Si","text":"Int","kind":"typeIdentifier"},{"text":")","kind":"text"}],"externalID":"s:10SwiftKyber9ExceptionO14cipherTextSizeyACSi_tcACmF"},"sections":[],"abstract":[{"text":"Wrong cipher text size","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"cipherTextSize","kind":"identifier"},{"kind":"text","text":"("},{"kind":"externalParam","text":"value"},{"text":": ","kind":"text"},{"text":"Int","kind":"typeIdentifier","preciseIdentifier":"s:Si"},{"kind":"text","text":")"}],"languages":["swift"]}]}],"identifier":{"url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/cipherTextSize(value:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"kind":"symbol","url":"\/documentation\/swiftkyber","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","type":"topic","title":"SwiftKyber","role":"collection","abstract":[{"text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard","type":"text"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"abstract":[{"type":"text","text":"The Kyber exceptions"}],"title":"Exception","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","type":"topic","role":"symbol","url":"\/documentation\/swiftkyber\/exception","navigatorTitle":[{"text":"Exception","kind":"identifier"}],"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Exception"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/cipherTextSize(value:)":{"kind":"symbol","url":"\/documentation\/swiftkyber\/exception\/ciphertextsize(value:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cipherTextSize","kind":"identifier"},{"kind":"text","text":"("},{"text":"value","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:Si","kind":"typeIdentifier","text":"Int"},{"kind":"text","text":")"}],"role":"symbol","title":"Exception.cipherTextSize(value:)","type":"topic","abstract":[{"text":"Wrong cipher text size","type":"text"}],"identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/cipherTextSize(value:)"}}}